### ISSUE TITLE
Out-GridView consistency crashing shell on 7.3.6

### ISSUE THREAD

**Issue by StevenBucher98** [PowerShell/PowerShell#20016]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://learn.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

I am experiencing problems with `Out-GridView` crashing whenever I try to use it. 

### Expected behavior

```console
PS C:\Users\stevenbucher.REDMOND> ls | Out-GridView
Opens Outgrid view
```

### Actual behavior

```console
PS C:\Users\stevenbucher.REDMOND> ls | Out-GridView

An error has occurred that was not properly handled. Additional information is shown below. The PowerShell process will exit.
Unhandled exception. System.TypeLoadException: Could not load type 'MS.Internal.SecurityCriticalDataForSet`1' from assembly 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.
   at Microsoft.Management.UI.Internal.OutGridViewWindow.<>c__DisplayClass16_0.<StartWindow>b__0(Object <p0>)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
Out-GridView: Exception has been thrown by the target of an invocation.

[process exited with code 3762504530 (0xe0434352)]
```

### Error details

```console
This is the error I get when I run Get-Error after restarting the shell, very confused why chocolately is mentioned:
PS C:\Users\stevenbucher.REDMOND> Get-Error

Exception             :
    Type                 : System.Management.Automation.ItemNotFoundException
    ErrorRecord          :
        Exception             :
            Type    : System.Management.Automation.ParentContainsErrorRecordException
            Message : Cannot find path 'C:\ProgramData\chocolatey\license\chocolatey.license.xml' because it does not
exist.
            HResult : -2146233087
        TargetObject          : C:\ProgramData\chocolatey\license\chocolatey.license.xml
        CategoryInfo          : ObjectNotFound: (C:\ProgramData\choc…colatey.license.xml:String) [],
ParentContainsErrorRecordException
        FullyQualifiedErrorId : PathNotFound
    ItemName             : C:\ProgramData\chocolatey\license\chocolatey.license.xml
    SessionStateCategory : Drive
    TargetSite           :
        Name          : ExpandMshGlobPath
        DeclaringType : System.Management.Automation.LocationGlobber, System.Management.Automation, Version=7.3.6.500,
Culture=neutral, PublicKeyToken=31bf3856ad364e35
        MemberType    : Method
        Module        : System.Management.Automation.dll
    Message              : Cannot find path 'C:\ProgramData\chocolatey\license\chocolatey.license.xml' because it does
not exist.
    Source               : System.Management.Automation
    HResult              : -2146233087
    StackTrace           :
   at System.Management.Automation.LocationGlobber.ExpandMshGlobPath(String path, Boolean allowNonexistingPaths,
PSDriveInfo drive, ContainerCmdletProvider provider, CmdletProviderContext context)
   at System.Management.Automation.LocationGlobber.ResolveDriveQualifiedPath(String path, CmdletProviderContext
context, Boolean allowNonexistingPaths, CmdletProvider& providerInstance)
   at System.Management.Automation.LocationGlobber.GetGlobbedMonadPathsFromMonadPath(String path, Boolean
allowNonexistingPaths, CmdletProviderContext context, CmdletProvider& providerInstance)
   at System.Management.Automation.LocationGlobber.GetGlobbedProviderPathsFromMonadPath(String path, Boolean
allowNonexistingPaths, CmdletProviderContext context, ProviderInfo& provider, CmdletProvider& providerInstance)
   at System.Management.Automation.SessionStateInternal.GetItem(String[] paths, CmdletProviderContext context)
   at Microsoft.PowerShell.Commands.GetItemCommand.ProcessRecord()
TargetObject          : C:\ProgramData\chocolatey\license\chocolatey.license.xml
CategoryInfo          : ObjectNotFound: (C:\ProgramData\choc…colatey.license.xml:String) [Get-Item],
ItemNotFoundException
FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetItemCommand
InvocationInfo        :
    MyCommand        : Get-Item
    ScriptLineNumber : 66
    OffsetInLine     : 20
    HistoryId        : 1
    ScriptName       : C:\ProgramData\chocolatey\helpers\ChocolateyTabExpansion.ps1
    Line             : $licenseFile = Get-Item -Path "$env:ChocolateyInstall\license\chocolatey.license.xml"
-ErrorAction Stop

    PositionMessage  : At C:\ProgramData\chocolatey\helpers\ChocolateyTabExpansion.ps1:66 char:20
                       + … censeFile = Get-Item -Path "$env:ChocolateyInstall\license\chocolatey …
                       +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    PSScriptRoot     : C:\ProgramData\chocolatey\helpers
    PSCommandPath    : C:\ProgramData\chocolatey\helpers\ChocolateyTabExpansion.ps1
    InvocationName   : Get-Item
    CommandOrigin    : Internal
ScriptStackTrace      : at <ScriptBlock>, C:\ProgramData\chocolatey\helpers\ChocolateyTabExpansion.ps1: line 66
                        at <ScriptBlock>, C:\ProgramData\chocolatey\helpers\chocolateyProfile.psm1: line 27
                        at <ScriptBlock>, \\Mac\Home\Documents\PowerShell\Microsoft.PowerShell_profile.ps1: line 148
                        at <ScriptBlock>, <No file>: line 1
```

### Environment data

```powershell
PS C:\Users\stevenbucher.REDMOND> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.3.6
PSEdition                      Core
GitCommitId                    7.3.6
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by sba923** [comment]
What's special about your 7.3.6 setup? I use `Out-GridView` day in, day out, never got a crash. I'm deploying from MSI.

**Comment by StevenBucher98** [comment]
I am not sure, its a parallels Windows VM so maybe something to do with shared file system

**Comment by mklement0** [comment]
I see the same problem on my W11 machine (VMWare Fusion VM running ARM64 Windows), but no longer in PowerShell 7.4.0-preview.4.

As for using `Get-Error`: it doesn't work across sessions, so what you're seeing is probably unrelated to the original crash.

**Comment by StevenBucher98** [comment]
Fixed in 7.4 previews
